### PR TITLE
@ #533 | should not display log tracing when the log level is not debug

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #533 
please check this PR, thank you

### CODE REVIEW

**Comment by hieptranquoc** [comment]
example for not debug log level:
```
[TeracyDev::Location::GitSynch][INFO]: cd /Users/hiep/k8s-dev/lib/../extensions && git clone git@github.com:hieptranquoc/teracy-dev-core.git 
Cloning into 'teracy-dev-core'...
Enter passphrase for key '/Users/hiep/.ssh/id_rsa': 
remote: Enumerating objects: 9, done.
remote: Counting objects: 100% (9/9), done.
remote: Compressing objects: 100% (9/9), done.
remote: Total 365 (delta 1), reused 3 (delta 0), pack-reused 356
Receiving objects: 100% (365/365), 64.07 KiB | 51.00 KiB/s, done.
Resolving deltas: 100% (154/154), done.
```
example for debug log level:
```
[TeracyDev::Location::GitSynch][DEBUG]: update git remote urls for /Users/hiep/k8s-dev/lib/../extensions/teracy-dev-core
/Users/hiep/k8s-dev/lib/teracy-dev/location/git_synch.rb:120:in `block in update_remote'
```

File: lib/teracy-dev/logging.rb [hunk 1]
```diff
@@ -60,11 +60,13 @@ def self.configure_logger_for(classname)
             log = TeracyDev::Common.yellow(log)
           when "INFO"
             log = TeracyDev::Common.green(log)
+          else
+            # display log message here with tracing when log level is debug only
```
Comment (datphan) [inline lib/teracy-dev/logging.rb]: should inside `when "DEBUG"` for more specific

Comment (hieptranquoc) [reply lib/teracy-dev/logging.rb]: i've updated, please check again!
```suggestion
            when "DEBUG"
```

Comment (hieptranquoc) [reply lib/teracy-dev/logging.rb]: @datphan

**Review by datphan — APPROVED** [review]

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**